File: 250101/숫자의 증감/increasing-and-decreasing-numbers.py
c, n1 = input().split()
n = int(n1) 

if c == 'A':
    ans = 1
    for _ in range (n):
        print(ans, end=" ")
        ans += 1
else:
    ans = n
    for _ in range(n):
        print(ans, end=" ")
        ans -= 1

File: 250211/숫자의 증감/increasing-and-decreasing-numbers.js
const fs = require("fs");
let [c, n] = fs.readFileSync(0).toString().trim().split(" ")

n = Number(n)

let ans =""

if (c === "A") {
    for (let i = 1; i <= n; i++) {
        ans += (i + " ")
    }
} else if (c === "D") {
    for (let i = n; i >= 1; i -= 1) {
        ans += (i + " ")
    }
}
console.log(ans);